### PR TITLE
Fix nodes count in local query runner

### PR DESCRIPTION
**Pull request by kokosing**
Fix nodes count in local query runner

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java [hunk 1]
```diff
@@ -443,7 +447,7 @@ public void close()
     @Override
     public int getNodeCount()
     {
-        return 1;
+        return nodesCount;
```
Comment (findepi) [inline presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java]: what's the meaning of `getNodeCount()`?

File: presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java [hunk 2]
```diff
@@ -335,8 +339,8 @@ private LocalQueryRunner(Session defaultSession, FeaturesConfig featuresConfig,
         this.statsCalculator = new SelectingStatsCalculator(
                 new CoefficientBasedStatsCalculator(metadata),
                 ServerMainModule.createNewStatsCalculator(metadata, new ScalarStatsCalculator(metadata), new SemiJoinStatsCalculator()));
-        this.costCalculator = new CostCalculatorUsingExchanges(this::getNodeCount);
```
Comment (findepi) [inline presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java:338]: good catch

Comment (findepi) [reply presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java:338]: i applied this as a fixup to `Support using a fake node count for unit tests`

File: presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java (hunk 1 above)
Comment (kokosing) [reply presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java]: IMO the same as as passing number of nodes count to LocalQueryRunner, to mock number of nodes. Let me see who is using this.

Comment (kokosing) [reply presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java]: I cannot remove that it is used in ATQ

**Comment by findepi** [comment]
The probably most important part (changes in `LocalQueryRunner.java`) were cherried into CBO epic. 
Are the other changes important, or should we close this?

**Pull request closed without merging**